.github/workflows/jira-issue-transfer.yml

```
@@ -31,20 +31,28 @@ jobs:
             JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
             GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         - name: Jira Create Bug
-          if: github.event.label.name == 'ticketed' && !contains(github.event.issue.labels.*.name, 'ticketed') && contains(github.event.issue.labels.*.name, 'bug') 
+          id: bug
+          if: github.event.label.name == 'ticketed'  && contains(github.event.issue.labels.*.name, 'bug') 
           uses: atlassian/gajira-create@v3
           with:
             project: BP
             issuetype: 'Bug Report'
             summary: ${{ github.event.issue.title }}
             description: "Github Issue Link: ${{ github.event.issue.html_url}} \r\n ${{ github.event.issue.body }}"
             fields: '{"labels":["GitHubReport"]}'
-        - name: Jira Create Enhancement        
-          if: github.event.label.name == 'ticketed' && !contains(github.event.issue.labels.*.name, 'ticketed') && contains(github.event.issue.labels.*.name, 'enhancement')
+
+        - name: Jira Create Enhancement
+          id: enhancement
+          if: github.event.label.name == 'ticketed'  && contains(github.event.issue.labels.*.name, 'enhancement')
           uses: atlassian/gajira-create@v3 
           with:
             project: BP
             issuetype: 'Product Feature'
             summary: ${{ github.event.issue.title }}
             description: "Github Issue Link: ${{ github.event.issue.html_url}} \r\n ${{ github.event.issue.body }}"
-            fields: '{"labels":["GitHubReport"]}'
\ No newline at end of file
+            fields: '{"labels":["GitHubReport"]}'
+            
+        - name: Read Ticket Number
+          id: Ticket
+          echo "Ticket Number: ${{ steps.enhancement.outputs.issue }} ${{ steps.bug.outputs.issue }}"
```

Each of the steps is basically an array of options, so you can't put code directly in line with it. The text of the code would need to be passed as the value of the `run` key like this
```suggestion
          run: echo "Ticket Number: ${{ steps.enhancement.outputs.issue }} ${{ steps.bug.outputs.issue }}"
```
or in multiline/block syntax like this:
```suggestion
          run: |
            echo "Ticket Number: ${{ steps.enhancement.outputs.issue }} ${{ steps.bug.outputs.issue }}"
```